examples/visualize_features.py

```diff
@@ -163,14 +187,28 @@ def main(rank: int = 0, world_size: int = 1):
             with torch.autocast(device.type, dtype=torch.bfloat16):
                 p_images = preprocessor(images)
 
-                output = model(p_images)
-                if args.adaptor_name:
-                    all_feat = [
-                        output['backbone'].features,
-                        output[args.adaptor_name].features,
-                    ]
+                if args.intermediates:
```

Comment:
There's an interesting opportunity to explore using all of the intermediate features to establish the PCA directions btw. 

We do this in the video feature visualizer: https://github.com/NVlabs/RADIO/blob/main/examples/visualize_video_features.py#L140

I'm not sure if treating each slice together or independently is more intuitive.